config/initializers/figaro.rb

```diff
@@ -6,7 +6,6 @@
   'domain_name',
   'enable_identity_verification',
   'enable_test_routes',
-  'equifax_ssh_passphrase',
```

Comment:
fwiw, I intend to immediately create PRs against master for any changes that should be merged back. There should never be any 2-weeks-later-review of changes.